docs/APIReference-AtomicBlockUtils.md

```diff
@@ -24,3 +24,21 @@ insertAtomicBlock: function(
   character: string
 ): EditorState
 ```
+
+### moveAtomicBlockBefore
+
+```
+moveAtomicBlockBefore: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
+
+### moveAtomicBlockAfter
+
+```
+moveAtomicBlockAfter: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
```

Comment:
Start to like that last version more and more. My fingertips can't wait to implement this! Haha.